src/dbcp/commands/publish.py

```diff
@@ -58,6 +60,31 @@ def upload_parquet_directory_to_gcs(
         )
 
 
+def load_parquet_files_to_postgres(
+    output_bucket: storage.Bucket,
+    destination_blob_prefix: DataDirectoryLiteral,
+    version: str,
+):
+    """
+    Load Parquet files from GCS to production postgres db.
+
+    Args:
+        output_bucket: the GCS bucket containing the output Parquet files.
+        destination_blob_prefix: the prefix of the GCS blobs to load.
+        version: the version of the data to load.
+    """
+    engine = get_sql_engine(production=True)
+    for blob in output_bucket.list_blobs(prefix=f"{version}/{destination_blob_prefix}"):
+        if not blob.name.endswith(".parquet"):
+            continue
+        # get the blob filename without the extension
+        table_name = blob.name.split("/")[-1].split(".")[0]
+
+        # Read parquet then write to postgres
+        df = pd.read_parquet(blob.path)
```

Comment:
For me, `blob.path` looks like a binary stream path (`/b/dgm-outputs/o/7305b1eb-ed1a-403c-8649-a212471a838f%2Fdata_mart%2Fbr_election_data.parquet`) which doesn't work with `pd.read_parquet`. Seems like the path can be constructed from:

`path = f"gs://{output_bucket.name}/{blob.name}"`

which worked for me.

Or you could do something like:

`pd.read_parquet(io.BytesIO(blob.download_as_bytes()))`

but that seems less clean